frontend/src/routes/Credentials/CreateCredentialsCatalog.tsx

```diff
@@ -33,19 +32,31 @@ export function CreateCredentialsCatalog() {
 
   const cards = useMemo(() => {
     const newCards: ICatalogCard[] = [
-      ...orderedProviders
-        // does not display AWS s3 card
-        .filter(([id]) => id !== Provider.awss3)
-        .map(([provider, id]) => ({
-          id: id || provider,
-          icon: <AcmIcon icon={ProviderIconMap[provider]} />,
-          title: ProviderLongTextMap[provider],
-          onClick:
-            provider === Provider.aws
-              ? nextStep(NavigationPath.addAWSType)
-              : nextStep(getTypedCreateCredentialsPath(provider)),
-        })),
+      ...orderedProviders.map(([provider, id]) => ({
+        id: id || provider,
+        icon: <AcmIcon icon={ProviderIconMap[provider]} />,
+        title: ProviderLongTextMap[provider],
+        onClick:
+          provider === Provider.aws
+            ? nextStep(NavigationPath.addAWSType)
+            : nextStep(getTypedCreateCredentialsPath(provider)),
+      })),
     ]
+    newCards.forEach((card) => {
+      if (card.id === 'rhv') {
+        ;(card.alertTitle = t('Deprecated host platform')),
+          (card.alertVariant = 'warning'),
+          (card.alertContent = (
+            <>
+              {t(
+                'Red Hat Virtualization is deprecated as a host platform for OpenShift 4.13 and will be removed in the next release.'
+              )}
+              <br />
+              <ViewDocumentationLink doclink={DOC_LINKS.RHV_DEPRECATION} />
+            </>
+          ))
+      }
+    })
```

Comment:
Personally I would do this during the map rather than looping through again, and base it on provider rather than ID. Something like:
```
...(provider === Provider.redhatvirtualization ? {alertTitle: ..., alertVariant: ..., ...} : {})
```